src/app/store.ts

```diff
@@ -1,10 +1,21 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer from '../features/users/usersSlice';
+import authorReducer from '../features/author/authorSlice';
+import postsReducer from '../features/posts/postsSlice';
+import selectedPostReducer from '../features/selectedPost/selectedPostSlice';
+// eslint-disable-next-line import/no-cycle
+import commentsReducer from '../features/comments/commentsSlice';
```

Comment:
try to fix it
Your imports was cycled
```suggestion
import commentsReducer from '../features/comments/commentsSlice';
```

Reply:
fixed